05_debug/sysfs_task/Makefile

```diff
@@ -0,0 +1,11 @@
+obj-m += 05_debug_sysfs.o
+
+BUILD_KERNEL := ~/repos/busybox/_install/lib/modules/4.19.177/build
```

Comment:
Just FYI: It is a good practice to point here to your build linux-stable/ folder since you may update you kernel and  kernel vesion is hardcoded here